docs/rst/user_guide/atmosphere/heterogeneous.rst

```diff
@@ -0,0 +1,194 @@
+number of layers the atmosphere is divided into:
+
+.. code::
+
+   from eradiate import unit_context_config as ucc
+   ucc.override({"length": "km"}):
+
+   atmosphere = eradiate.scenes.atmosphere.HeterogeneousAtmosphere(
+       profile={
+           "type": "us76_approx",
+           "height": 120.,
+           "n_layers": 120,
+       },
+   )
+
+The corresponding atmosphere is 120 km high and is divided into 120 layers
+(each 1 km thick).
+
+.. note::
+
+   The layers are of equal thickness.
+
+.. note::
+
+   When you set the ``height`` parameter of the ``us76_approx`` profile object,
+   the corresponding atmosphere ``toa_altitude`` is automatically adjusted to
+   that value.
+   In fact, you cannot set both the ``profile`` and ``toa_altitude`` attributes
+   of the :class:`~eradiate.scenes.atmosphere.HeterogeneousAtmosphere` class.
+   The following code, for example, would raise an exception:
+
+   .. code:: python
+
+      atmosphere = eradiate.scenes.atmosphere.HeterogeneousAtmosphere(
+          toa_altitude=100,
+          profile={
+              "type": "us76_approx",
+              "height": 120.,
+              "n_layers": 120,
+          },
+      )
+
+   For more detail, refer to the
+   :class:`~eradiate.radprops.rad_profile.US76ApproxRadProfile`
+   reference documentation.
+
+By default, the width of the heterogeneous atmosphere is set to 1000 km.
+This width guarantees the absence of edge effects in simulations where the
+sensor is a radiance meter array placed at the top of the atmosphere and looking
+down with a zenith angle varying from 0 to 75°.
+Above 75°, the measured values start to be influenced by the fact that the
+horizontal size of the atmosphere is finite.
+For accurate results above 75°, consider increasing the atmosphere width, using
+the ``width`` attribute:
+
+.. code:: python
+
+   atmosphere = eradiate.scenes.atmosphere.HeterogeneousAtmosphere(
+       width=1e4,
+       profile={
+           "type": "us76_approx",
+           "height": 120.,
+           "n_layers": 120,
+       },
+   )
+
+array
+~~~~~
+
+To create a heterogeneous atmosphere based on the
+:class:`array <eradiate.radprops.rad_profile.ArrayRadProfile>` radiative
+properties profile type, use:
+
+.. code:: python
+
+   import numpy as np
+
+   atmosphere = eradiate.scenes.atmosphere.HeterogeneousAtmosphere(
+       profile={
+           "type": "array",
+           "sigma_t_values": np.array([1e-5, 1e-6, 1e-7, 1e-8]).reshape(1, 1, 4),
+           "albedo_values": np.array([.95, .97, .99, 1.]).reshape(1, 1, 4),
+           "height": 100.,
+       },
+   )
+
+The corresponding atmosphere is 100 km high and is divided into 4 layers
+(each 25 km thick).
+The first values in the  ``sigma_t_values`` and ``albedo_values`` arrays
+correspond to the bottom layer of the atmosphere.
+
+.. _sec-atmosphere-heterogeneous-kernel_volume_data_files:
+
+Kernel volume data files
+------------------------
```

Comment:
Please mention this as advanced content (this is more for maintainers, actually).